### PR TITLE
Remove unused mesosTaskID constant

### PR DESCRIPTION
**Pull request by gnufied**

### CODE REVIEW

File: src/fullerite/collector/docker_stats.go [hunk 1]
```diff
@@ -14,7 +14,6 @@ import (
 )
 
 const (
-	mesosTaskID      = "MESOS_TASK_ID"
 	endpoint         = "unix:///var/run/docker.sock"
 	serviceNameLabel = "SERVICE_NAME"
```
Comment (tsheasha) [inline src/fullerite/collector/docker_stats.go]: This is also not used no?

Comment (gnufied) [inline src/fullerite/collector/docker_stats.go]: yep. fixed

**Comment by tsheasha** [comment]
+1

**Comment by tsheasha** [comment]
@dichiarafrancesco objections?

**Comment by zachm** [comment]
+1

Should get signoff from @dichiarafrancesco though.

**Comment by dichiarafrancesco** [comment]
+1

**Pull request merged**